pkg/client/cache/reflector.go

```diff
@@ -265,6 +265,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 			switch err {
 			case io.EOF:
 				// watch closed normally
+				glog.V(1).Infof("%s: Watch for %v closed normally (with EOF)", r.name, r.expectedType)
```

Comment:
We put other things like this at 4 - you can always enable these by default
in the cluster for just this package

On Thu, Feb 4, 2016 at 11:38 AM, Prashanth B <notifications@github.com>
wrote:

> In pkg/client/cache/reflector.go
> <https://github.com/kubernetes/kubernetes/pull/20637#discussion_r51898886>
> :
>
> > @@ -265,6 +265,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
> >  			switch err {
> >  			case io.EOF:
> >  				// watch closed normally
> > +				glog.V(1).Infof("%s: Watch for %v closed normally (with EOF)", r.name, r.expectedType)
>
> I'm fine adding it at V(4).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20637/files#r51898886>.
>